src/components/button.tsx

```diff
@@ -8,21 +8,29 @@ export default function Button(props: {
   ariaLabel?: string;
 }) {
   const classNames = `shadow-sm shadow-wai-gray bg-pure-white text-xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center`;
+  
+  // To allow for \n linebreaks
+  const formattedText = props.text.split('\n').map((line, index) => (
```

Comment:
have you considered https://stackoverflow.com/questions/39325414/line-break-in-html-with-n

Reply:
i did, it didn't work, i did again, it work. FML.